doc/modules/linear_model.rst

```diff
@@ -897,6 +897,169 @@ to warm-starting (see :term:`Glossary <warm_start>`).
     .. [9] `"Performance Evaluation of Lbfgs vs other solvers"
             <http://www.fuzihao.org/blog/2016/01/16/Comparison-of-Gradient-Descent-Stochastic-Gradient-Descent-and-L-BFGS/>`_
 
+.. _Generalized_linear_regression:
+
+Generalized Linear Regression
+=============================
+
+Generalized Linear Models (GLM) extend linear models in two ways
+[10]_. First, the predicted values :math:`\hat{y}` are linked to a linear
+combination of the input variables :math:`X` via an inverse link function
+:math:`h` as
+
+.. math::    \hat{y}(w, x) = h(x^\top w) = h(w_0 + w_1 x_1 + ... + w_p x_p).
+
+Secondly, the squared loss function is replaced by the unit deviance :math:`d`
+of a reproductive exponential dispersion model (EDM) [11]_. The minimization
+problem becomes
+
+.. math::    \min_{w} \frac{1}{2 \sum_i s_i} \sum_i s_i \cdot d(y_i, \hat{y}(w, x_i)) + \frac{\alpha}{2} ||w||_2
```

Comment:
The other formulas (e.g. for ridge and lasso) multiply by `alpha`, not `alpha / 2`.

Is the use of alpha consistent across  the implementations??

Reply:
I think it is 100% consistent to the L2 penalty of ElasticNet: `TweedieRegressor(power=0, link='identity', alpha=42)` is the same as `ElasticNet(alpha=42, l1_ratio=0)`). But there is the well known difference to Ridge, where the L2 penalty gets smaller the larger n_samples. (I prefer the ElasticNet version and we aim at adding L1 to GLMs.)